Dockerfile

```diff
@@ -55,7 +54,8 @@ RUN apt-get -y install git make gcc libperl-dev npm curl && \
     rm -rf /usr/local/lib/node_modules && \
     apt-get autoremove -y && \
     apt-get autoclean && \
-    rm -rf ~/.cpanm
+    rm -rf ~/.cpanm && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
This looks redundant... would be curious whether the result is significant of removing this at all, but seems like it has already been done twice...